MonMaraicher/Features/FarmerDetail/FarmerDetailsViewModel.swift

```diff
@@ -10,36 +10,25 @@ import MapKit
 struct FarmerDetailsViewModel {
 
     let title: String
+    let phoneNumber: String?
+    let productions: [Productions]
     let coordinate: CLLocationCoordinate2D
-    let farmerImages: [String]
     let markerSystemImageName: String
     let directionButtonTitle: String
     let address: String
+    let typeAddress: [String]
     let city: String
 
-    init(farmer: Farmer) {
-        self.title = farmer.name.capitalized
-        self.coordinate = .init(latitude: farmer.location.latitude,
-                                longitude: farmer.location.longitude)
-        self.farmerImages = [farmer.images.farmer1,
-                           farmer.images.farmer2,
-                           farmer.images.farmer3,
-                           farmer.images.farmer4,
-                           farmer.images.farmer5,
-                           farmer.images.farmer6]
-        self.address = Self.formatAddress(farmer.location.address).capitalized
-        self.city = farmer.location.address.city.capitalized
+    init(farmer: Farmer, address: AdressesOperateurs) {
```

Comment:
`AdressesOperateurs` --> `AdressesOperators` --> `OperatorsAdresses`

Reply:
It's simple!